source/expressions.tex

```diff
@@ -2819,12 +2819,12 @@
     postfix-expression \opt{\terminal{-> template}} id-expression\br
     postfix-expression \terminal{++}\br
     postfix-expression \terminal{-{-}}\br
-    \terminal{dynamic_cast <} type-id \terminal{> (} expression \terminal{)}\br
-    \terminal{static_cast <} type-id \terminal{> (} expression \terminal{)}\br
-    \terminal{reinterpret_cast <} type-id \terminal{> (} expression \terminal{)}\br
-    \terminal{const_cast <} type-id \terminal{> (} expression \terminal{)}\br
-    \terminal{typeid (} expression \terminal{)}\br
-    \terminal{typeid (} type-id \terminal{)}
+    \keyword{dynamic_cast} \terminal{<} type-id \terminal{> (} expression \terminal{)}\br
```

Comment:
Hmm. Somewhat pre-existing, but... I believe this will cause us to use a roman italic (narrow) space between the `dynamic_cast` and the `<`, but a teletype space between the `>` and `(`. We should think about what we actually want here; I'm inclined to prefer the roman italic space.

I'll file this as a separate issue.

Reply:
Filed as #2788